Makefile

```diff
@@ -12,10 +12,24 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-all: verify test
+GOBIN=$(shell pwd)/hack/tools/bin
+
+all: install-tools verify golangci-lint test
 
 dep:
 	go mod tidy
+	cd hack/tools && go mod tidy
+
+install-tools: $(GOBIN)
+	cd hack/tools \
+		&& GOBIN=$(GOBIN) go install github.com/golangci/golangci-lint/cmd/golangci-lint \
+		&& go build -o "${GOBIN}/logcheck.so" -buildmode=plugin sigs.k8s.io/logtools/logcheck/plugin
```

Comment:
You don't need golangci-lint to run logcheck.

Instead you can do:
```
go install sigs.k8s.io/logtools/logcheck@v0.7.0
logcheck -check-contextual ./...
```

I'm just using `/tmp` as placeholder.

Reply:
@bells17 can you please update the makefile with the suggestion?